### ISSUE TITLE
Additive Blendshapes go over 100

### ISSUE THREAD

**Issue by DFliyerz** [lyuma/Av3Emulator#45]
When using additive layers, blendshapes can be forced over 100%. In VRChat this doesn't happen, and blendshape values will cap out at 100. It'd be nice if the emulator replicated this.

**Comment by lyuma** [comment]
This type of issue can occur if you reset the avatar or use the "Debug Duplicate Animator" setting to anything other than Base.

I just released an update 2.9.8 which should mitigate one of the causes of this issue.

I would also suggest that you make sure to set "Animator to Debug" from the "Avatar 3.0 Emulator Control" object before entering play mode, to guarantee that it will not reset your avatar.

Finally, does it happen if your Debug Animator settings are set to Base?

**ALSO!!** Any attempt to edit the animator, or any animations will also reset the Animator. This "feature" is builtin to Unity, and it will cause this problem as well. I don't know of any fix to this. It's a bug in unity itself.

However, I did throw in one feature which may workaround the issue if you want to edit your animator and test additive layers without leaving play mode: Every time you make a change and things get messed up, click the "Create Non-Local Clone". Then the newly created remote clone should look correct even if the others are wrong.

Let me know if I didn't solve your issue. Sorry Unity's Animator system is so glitchy and buggy in the editor. All these confusing workarounds I had to create still only half-solve some of the problems.
